schemas/audiences/segment.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/audiences/segment",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Segment Assignment",
+  "type": "object",
+  "description":
+    "",
+  "definitions": {
+    "segment": {
+      "properties": {
+        "@id": {
+          "title": "Identity",
+          "type": "string",
+          "format": "uri",
+          "description": "Unique identity of the segment."
+        },
+        "xdm:endUserIDs": {
+          "title": "End User IDs",
+          "$ref": "https://ns.adobe.com/xdm/context/enduserids",
+          "description":
+            "Condensed, normalized encapsulation of an end user identifier that the segment is assigned to."
```

Comment:
Ok, here is my interpretation: a person (`Profile`) belongs to a somehow defined group of people (`Audience`). This declaration of membership is made in this schema, as a "Segment Assignment".

So, what's the role of `EndUserIDs`?

Some applications that perform an assignment perform the assignment based on `EndUserIDs`, so it makes sense to track this.

If my interpretation is correct, we need to rephrase the description to something like:

> The list of `EndUserIDs` that the segment assignment is based on.